src/stores/SearchStore.ts

```diff
@@ -113,7 +113,7 @@ export class SearchStore {
 		this.init();
 
 		autorun(() => {
-			this.currentSearch = this.searchHistory[this.currentIndex] || null;
+			this.currentSearch = this.currentIndex > 0 ? this.searchHistory[this.currentIndex] : null;
```

Comment:
What type of warning does it fix?
Why current index has to be greater than 0 ? we ignore 0 index element?

that warning?
Attempt to read an array index (0) that is out of bounds (0). 

I think it's about checking array length first before accessing it.